### PR TITLE
[bugmon-win] Additional fixes to configuration paths

### PR DESCRIPTION
**Pull request by pyoor**

### CODE REVIEW

File: services/bugmon-win/launch.sh [hunk 1]
```diff
@@ -28,8 +28,8 @@ EOF
 
 retry python -m pip install git+https://github.com/MozillaSecurity/bugmon-tc.git
 
-ARTIFACT_DEST="$USERPROFILE/bugmon-artifacts"
-TC_ARTIFACT_ROOT="project/fuzzing/bugmon"
+ARTIFACT_DEST="./bugmon-artifacts"
+TC_ARTIFACT_ROOT="project\fuzzing\bugmon"
```
Comment (jschwartzentruber) [inline services/bugmon-win/launch.sh:32]: Are you sure about this? (`\` vs `/`) Even if it needs to be `\`, you probably need to double them or sh will interpret them.

```
$ a="test\fuzz\backup"
$ echo "$a"
test
    uzackup
$ 
```

File: services/bugmon-win/launch.sh [hunk 2]
```diff
@@ -17,8 +17,8 @@ retry_curl () { curl -sSL --connect-timeout 25 --fail --retry 5 -w "%{stderr}[do
 powershell -ExecutionPolicy Bypass -NoProfile -Command "Set-MpPreference -DisableScriptScanning \$true" || true
 powershell -ExecutionPolicy Bypass -NoProfile -Command "Set-MpPreference -DisableRealtimeMonitoring \$true" || true
 
-mkdir -p "$USERPROFILE/Local/autobisect/autobisect/"
-cat << EOF > "$USERPROFILE/Local/autobisect/autobisect/autobisect.ini"
+mkdir -p "$USERPROFILE/AppData/Local/autobisect/autobisect/"
+cat << EOF > "$USERPROFILE/AppData/Local/autobisect/autobisect/autobisect.ini"
```
Comment (jschwartzentruber) [inline services/bugmon-win/launch.sh:21]: You could replace `$USERPROFILE/AppData/Local` with `$LOCALAPPDATA`.

File: services/bugmon-win/launch.sh [hunk 3]
```diff
@@ -28,8 +28,8 @@ EOF
 
 retry python -m pip install git+https://github.com/MozillaSecurity/bugmon-tc.git
 
-ARTIFACT_DEST="$USERPROFILE/bugmon-artifacts"
-TC_ARTIFACT_ROOT="project/fuzzing/bugmon"
+ARTIFACT_DEST="./bugmon-artifacts"
```
Comment (jschwartzentruber) [inline services/bugmon-win/launch.sh:31]: You could use `$PWD` if you do want it to be absolute.

**Review by jschwartzentruber — CHANGES_REQUESTED** [review]

File: services/bugmon-win/launch.sh (hunk 1 above)
Comment (pyoor) [reply services/bugmon-win/launch.sh]: This was actually unintenional.  This path is a TC resource and should always use `/`.

File: services/bugmon-win/launch.sh (hunk 3 above)
Comment (pyoor) [reply services/bugmon-win/launch.sh]: Done.

**Review by jschwartzentruber — APPROVED** [review]

**Pull request merged**